docs/channels.md

```diff
@@ -0,0 +1,768 @@
+
+:channels &channel-action [%channel nest action]
+```
+
+Edit a post in a channel
+
+[essay](#essay) \| [id-post](#id-post)
+###### %post %del
+```hoon
+=nest [%chat our %mychat]
+=post-action [%del ~2023.1.1]
+=action [%post post-action]
+
+:channels &channel-action [%channel nest action]
+```
+
+Delete a post
+
+[id-post](#id-post)
+###### %post %add-react
+```hoon
+=nest [%chat our %mychat]
+=post-action [%add-react ~2023.1.1 our %':grinning:']
+=action [%post post-action]
+
+:channels &channel-action [%channel nest action]
+```
+
+Add a reaction to a post
+
+[react](#react)
+###### %post %del-react
+```hoon
+=nest [%chat our %mychat]
+=post-action [%del-react ~2023.1.1 our]
+=action [%post post-action]
+
+:channels &channel-action [%channel nest action]
+```
+
+Remove a reaction from a post
+
+[react](#react)
+###### %post %reply %add
+```hoon
+=nest [%chat our %mychat]
+=memo :*
+  ~[[%inline ~['howdy world!']]]
+  author=our
+  sent=now
+==
+=reply-action [%add memo]
+=post-action [%reply ~2023.1.1 reply-action]
+=action [%post post-action]
+:channels &channel-action [%channel nest action]
+```
+
+Add a reply to a post
+
+[id-post](#id-post) \| [post](#post)
+###### %post %reply %del
+```hoon
+=nest [%chat our %mychat]
+=reply-action [%del ~2023.2.2]
+=post-action [%reply ~2023.1.1 reply-action]
+=action [%post post-action]
+:channels &channel-action [%channel nest action]
+```
+
+Delete a reply from a post
+
+[id-post](#id-post) \| [id-reply](#id-reply)
+###### %post %reply %add-react
+```hoon
+=nest [%chat our %mychat]
+=reply-action [%add-react ~2023.2.2 our %':grinning:']
+=post-action [%reply ~2023.1.1 reply-action]
+=action [%post post-action]
+:channels &channel-action [%channel nest action]
+```
+
+Add a reaction to a reply
+
+[id-post](#id-post) \| [id-reply](#id-reply) \| [react](#react)
+###### %post %reply %del-react
+```hoon
+=nest [%chat our %mychat]
+=reply-action [%del-react ~2023.2.2 our]
+=post-action [%reply ~2023.1.1 reply-action]
+=action [%post post-action]
+:channels &channel-action [%channel nest action]
+```
+
+Delete a reaction from a reply
+
+[id-post](#id-post) | [id-reply](#id-reply)
+###### %view
+```hoon
+=nest [%chat our %mychat]
+=action [%view %grid]
+:channels &channel-action [%channel nest action]
+```
+
+Set the display format for a channel
+
+[view](#view-1)
+###### %sort
+```hoon
+=nest [%chat our %mychat]
+=action [%sort %alpha]
+:channels &channel-action [%channel nest action]
+```
+
+Set the sorting mechanism for a channel
+
+[sort](#sort-1)
+###### %set-order
+```hoon
+=nest [%chat our %mychat]
+=action [%set-order `~[~2023.1.1 ~2023.2.2]]
+:channels &channel-action [%channel nest action]
+```
+
+Set manual post ordering for a channel
+
+[arranged-posts](#arranged-posts)
+###### %add-writers
+```hoon
+=nest [%chat our %mychat]
+=action [%add-writers (sy ~[%role1 %role2])]
+:channels &channel-action [%channel nest action]
+```
+
+Enable writing to a channel for a certain set of roles
```

Comment:
Here and below, "allow" and "disallow" might be better wording than "enable" and "disable" respectively. More evocative of dealing with permissions.  
Maybe also just "certain roles".